pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -58,11 +64,28 @@ Examples:
 				newBuild = buildutil.GenerateBuildFromBuild(build)
 			}
 
+			var wg sync.WaitGroup
+			if streamLogs {
+				rd, err := client.Get().Namespace(namespace).Prefix("redirect").Resource("buildLogs").Name(buildName).Stream()
+				checkErr(err)
+				wg.Add(1)
+				go func(rd io.ReadCloser, out io.Writer) {
+					defer wg.Done()
+					defer rd.Close()
+
+					_, err = io.Copy(out, rd)
+					checkErr(err)
+				}(rd, out)
+
+			}
+
 			newBuild, err = client.Builds(namespace).Create(newBuild)
 			checkErr(err)
+			wg.Wait()
 			fmt.Fprintf(out, "%s\n", newBuild.Name)
 		},
 	}
 	cmd.Flags().String("from-build", "", "Specify the name of a build which should be re-run")
+	cmd.Flags().Bool("stream-logs", false, "Run a build and stream its logs")
```

Comment:
I am not very familiar with resources, just getting into them but regarding logging I believe it should be exposed as an interface (instead of a resource - buildLogs) which in turn should be implemented by any resource that needed logging. I am getting something wrong here?